app/src/androidTest/java/net/skyscanner/backpack/compose/starrating/BpkStarRatingTest.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package net.skyscanner.backpack.compose.starrating
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import net.skyscanner.backpack.compose.BpkSnapshotTest
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class BpkStarRatingTest : BpkSnapshotTest() {
+
+  @Test
+  fun ratingFullStar() = snap {
```

Comment:
can you include some tests for the rounding, size + hotels one as well? might be worth looking at the view version to see what is tested there